ansible/roles/test/templates/vnet_config.j2

```diff
@@ -0,0 +1,38 @@
+
+{% set vlan_id_list = [ 2001, 2002 ] %}
```

Comment:
I understand this describes the topology of the test and It corresponds to the one in our OneNote called "Two Vnets, one HSM per Vnet" (but w/ fewer VMs)
Additional tests that I think would make sense.
- have two servers/HSMs per Vnet
- vnet peering
- overlapping ip addresses across Vnets

Reply:
@albgonzpri , sure i'll see to extend the test to cover the above scenarios.

Reply:
Thanks!